src/functionalTest/java/uk/gov/hmcts/reform/locationrefdata/cameltest/LrdApplicationTest.java

```diff
@@ -84,16 +86,21 @@ void testTaskletSuccess() throws Exception {
     @Test
     @Sql(scripts = {"/testData/truncate-lrd.sql"})
     void testTaskletSuccessWithInsertAndTruncateInsertDay2() throws Exception {
-
         testInsertion();
+        setLrdCamelRouteToExecute(ROUTE_TO_EXECUTE);
+        setLrdFileToLoad(UPLOAD_ORG_SERVICE_FILE_NAME);
```

Comment:
I think it is better to have these in `@BeforEach`. instead of having in each `@Test`